community-edition/readme.md

```diff
@@ -1,48 +1,57 @@
+# Introduction
+
+Community Edition is a free tool designed to help developers deploy the full Hubs stack on cloud computing software of their choosing. Community Edition simplifies and automates most of the complex deployment process using Kubernetes, a containerized software orchestration system.
+
+Community Edition is designed for developers capable of working with the full Hubs stack and of navigating complex application infrastructure. With Community Edition, developers are responsible for designing, building, hosting, and maintaining their software throughout its lifetime. In order to create a production-ready version of Hubs similar to Hubs Cloud, Developers using Community Edition will need to implement additional features and customizations outside of those listed in this repo. See "Considerations for Production Environment".
+
+# Why Kubernetes
+
+Kubernetes is an industry standard for allowing developers to build, deploy, and scale application efficiently and reliably. Benefits to Kubernetes include:
+
+- [Portability, Extensibility, and Open Source](https://kubernetes.io/docs/concepts/overview/)
+- [Availability in many cloud environments](https://kubernetes.io/docs/setup/production-environment/turnkey-solutions/)
+- Many options for single server deployments (ie. minikube, k3s, microk8s, kind)
+
+# Prerequisites
+
+Before deploying the charts in this repo, you will need to choose and configure the following services...
+
+- A hosting service with a Kubernetes cluster to receive your Community Edition deployment spec
+- Kubernetes controls on your device
+- A DNS service to reach Hubs on a domain
+- Port to expose services to client
+  - TCP: 80, 443, 4443, 5349
+  - UDP: 35000 -> 60000
+- An SMTP service for login emails and accounts (for login emails, ie. [use gmail as smtp](https://support.google.com/a/answer/176600?hl=en))
+
+# Deploy to Kubernetes
+
+To deploy to your K8s cluster on your chosen hosting solution, follow these steps:
+
+- Add your chosen services into `render_hcce.sh` with a text editor
+- Run `bash render_hcce.sh && kubectl apply -f hcce.yaml`
```

Comment:
Run seems to be unecessary, for example https://code.visualstudio.com/docs/setup/linux

Reply:
I think my problem with the current formatting is that the commands are in bulleted list. The example you shared has specific formatting for code snippets which we don't currently have here